src/main/java/de/tum/in/www1/artemis/domain/UserGroup.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.in.www1.artemis.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "user_groups")
+public class UserGroup {
```

Comment:
This class seems to be unused, but it is used in an SQL query. Teamscale does not recognizes this.

Reply:
Do you nee the column and entity annotations if this is only used inside of a Query? I think you can also call directly constructors in a JPA-query.